services/order.py

```diff
@@ -0,0 +1,33 @@
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket, User
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None
+) -> Order:
+    with transaction.atomic():
+        order = Order.objects.create(user=User.objects.get(username=username))
+        if date:
+            order.created_at = date
+            order.save()
+        for ticket in tickets:
+            Ticket.objects.create(
+                movie_session_id=ticket["movie_session"],
+                order=order,
+                row=ticket["row"],
+                seat=ticket["seat"]
+            )
+        return order

```

Comment:
This is a code quality issue. Your function is doing too much. It's creating an order, optionally setting a date, and creating tickets. Consider breaking it up into smaller functions, each with a single responsibility. This will make your code easier to read, test, and maintain.